hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3368,6 +3368,13 @@
     <description>Time interval of the SST File filtering service from Snapshot.
     </description>
   </property>
+  <property>
+    <name>ozone.snapshot.dir.check.poll.timeout</name>
+    <value>5s</value>
```

Comment:
Let's keep this to a minute so that we do not have to worry about even a very large directory with lots of sst files.

Reply:
Changed it 20secs